src/Citezen-Model.package/CZBooklet.class/class/initialize.st

```diff
@@ -1,4 +1,5 @@
 initialization
 initialize 
+	self class initialize.
 	self initializeNeededFields.
 	self initializeOptionalFields.
```

Comment:
I actually forced the initialization because the methods initializeFieldKeysToRemove and initializeExtendedFieldKeys which are identical for all the entries are defined in CZEntry. How am i supposed to do the initialization without calling CZEntry's method? I'm using the list -extendedFieldKeys- in the method initializeOptionalFields of each Entry. Or is it done automatically ? I assumed the list would be nil

![CZEntry initialize](https://user-images.githubusercontent.com/73491885/205713705-8c0a5ec0-dca9-4080-b838-9e07fa2abb66.png)